src/cmd/builtin/overlay.c

```diff
@@ -39,7 +41,7 @@ static struct optparse_option status_opts[] = {
                " 2=show round-trip RPC times."
     },
     { .name = "timeout", .key = 't', .has_arg = 1, .arginfo = "FSD",
-      .usage = "Set RPC timeout (default none)",
+      .usage = "Set RPC timeout, 0=disable (default 5s)",
```

Comment:
Update default

Reply:
Good catch!  Forcing a push with that change and setting MWP.